webbpsf/roman.py

```diff
@@ -466,7 +466,7 @@ def __init__(self):
             Set to True or False to force using or not using the cold pupil mask,
             or to None for the automatic behavior.
         """
-        pixelscale = 110e-3  # arcsec/px, WFIRST-AFTA SDT report final version (p. 91)
+        pixelscale = 110e-3  # arcsec/px, Roman-AFTA SDT report final version (p. 91)
```

Comment:
super minor, but we can also cite https://roman.ipac.caltech.edu/sims/Param_db.html for this (it's the same value, but more current reference than the old WFIRST-AFTA SDT report.)